dra-evolution/testdata/classes.yaml

```diff
@@ -1,162 +1,208 @@
 # Any Foozer device, assuming all are managed by the same driver.
-apiVersion: devmgmtproto.k8s.io/v1alpha1
-kind: DeviceClass
+apiVersion: resource.k8s.io/v1alpha2
+kind: ResourceClass
 metadata:
   name: example.com-foozer
-  labels:
-    vendor: example.com
-    device-type: gpu
-spec:
-  driver: example.com-foozer
+request:
```

Comment:
I don't understand "request" in a class?

Reply:
These are "requirements for requests". See https://github.com/kubernetes-sigs/wg-device-management/pull/14/files#r1611132210